bedrock/privacy/views.py

```diff
@@ -42,7 +42,7 @@ def get_legal_doc(self):
 
 
 class FirefoxPrivacyDocView(PrivacyDocView):
-    template_name = 'privacy/notices/firefox.html'
+    template_name = 'privacy/notices/firefox-quantum.html'
```

Comment:
I think this needs to extend "privacy/notices/base-notice-paragraphs.html" not the quantum template.

Reply:
(I'm still looking, but I didn't want anyone to R+ before I finish looking).

Reply:
Ah you're right, these have the expando-collapso sections. Updating.